src/components/featured-products-section/featured-products-section.module.scss

```diff
@@ -31,3 +33,42 @@
     gap: 4px;
     width: 100%;
 }
+
+@media (max-width: $desktop-width) {
+    .root {
+        padding-left: 4%;
+        padding-right: 4%;
+    }
+
+    .headerDescription {
+        font-size: 16px;
+    }
+}
+
+@media (max-width: $tablet-width) {
+    .root {
+        padding-top: 40px;
+        padding-bottom: 40px;
+    }
+
+    .header {
+        display: block;
+        font-size: 30px;
+        margin-bottom: 40px;
+    }
+
+    .headerTitle {
+        font-size: 30px;
+        margin-bottom: 16px;
+    }
+
+    .headerDescription {
+        font-size: 14px;
+    }
+
+    .productsRow {
+        grid-template-columns: repeat(auto-fit, minmax(min(135px, 100%), 1fr));
```

Comment:
`repeat(auto-fit, minmax(min(135px, 100%), 1fr))`

Can you explain this?

Reply:
Right. The expression is good for wix template where card `width` (here 135px) is dynamic and can be bigger than `grid` width. In this case `min(--card-width, 100%)` provides min column width if `--card-width < grid width`
We have no grid less than 135px of witdh, so the expression can be simplified to `repeat(auto-fit, minmax(135px, 1fr));`